mlir/utils/jenkins/Jenkinsfile

```diff
@@ -256,7 +256,7 @@ pipeline {
                             steps {
                                 buildProject('check-rocmlir',
                                              """-DROCMLIR_DRIVER_PR_E2E_TEST_ENABLED=0
-                                             -DROCMLIR_DRIVER_XDLOPS_TEST_ENABLED=${params.canXdlops ? '1' : '0'}
+                                             -DROCMLIR_GEN_FEATURES=${params.canXdlops ? '-mfma=on -atomic_add=on' : '-mfma=off --atomic_add=off'}
```

Comment:
Given that we're passing `%arch`, we don't _need_ to set this.
I can see leaving it in, just so that when the multi-feature PR lands, we have the hook to customize it, but we don't actually have to set `-mfma=on`